File: year2020/month09/day0908/gebal_0908.c
#include<stdio.h>

void proc(int *a, int *b) {
	if (*a > *b) {
		*a /= 2;
		*b *= 2;
	}
	else {
		*a *= 2;
		*b /= 2;
	}
}

int main() {
	int a, b;
	scanf("%d %d", &a, &b);
	proc(&a, &b);
	printf("%d %d", a, b);
}

File: year2020/month09/day0908/gebal_0908.py
n = list(map(int, input().split(' ')))

def proc(n):
    if n[0] > n[1]:
        n[0] /= 2
        n[1] *= 2
    else:
        n[0] *= 2
        n[1] /= 2

proc(n)
print(int(n[0]), int(n[1]))
